proguard.cfg

```diff
@@ -1,83 +0,0 @@
--optimizationpasses 5
```

Comment:
Why did you delete the proguard file altogether? Please keep it.

Reply:
There were 2 instances of this file with only a small difference in formatting, so identical in content.
One under the root directory and one under the app module, so I removed the one in the root, it looked like that it was not used. `build.gradle` in app module references the one next to it.

Reply:
ah, right.